src/v3/jest.config.js

```diff
@@ -18,6 +18,9 @@ const REPORT_DIR = '<rootDir>/build2/reports/unit';
 const esModules = [
   '@okta/odyssey-design-tokens',
   '@okta/odyssey-react-mui',
+  '@adobe/leonardo-contrast-colors',
+  'apca-w3',
+  'colorparsley'
```

Comment:
Latest version of Leonardo + these 2 new dependencies it added are all not pre-compiled

Reply:
Have you checked this branch on IE11?
Should new 2 deps be added to `npmRequiresTransform`?
https://github.com/okta/okta-signin-widget/blob/4911138be04100fb2451979abaee32ee41236f81/src/v3/webpack.common.config.ts#L97

Reply:
Good call out, will likely have to add and then will verify on SauceLabs

Reply:
Yes, they were required. I've added them 👍